modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -43,6 +49,19 @@ class Configuration(object):
       then all undeclared properties received by the server are injected into the
       additional properties map. In that case, there are undeclared properties, and
       nothing to discard.
+    :param disable_client_side_validation (string/None): Comma-separated list of
```

Comment:
How about naming this `disabled_client_side_validations` and making its type string only?
That way "" means that no validations are omitted and the type is always string.